frontend/src/app/libs/event-description/store/event-description/event-description.effects.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@angular/core';
+import { Actions, ofType, Effect } from '@ngrx/effects';
+import { catchError, map, switchMap } from 'rxjs/operators';
+import { of, Observable } from 'rxjs';
+import {
+	EventDescriptionActionTypes,
+	LoadEventDescription,
+	LoadEventDescriptionsSuccess,
+	LoadEventDescriptionsFailure,
+} from './event-description.actions';
+import { Action } from '@ngrx/store';
+import { EventsRepository } from 'src/app/libs/landing/services/events-repository.service';
+import { EventDto } from 'src/app/libs/landing/models/event-dto';
+import { EventConverter } from 'src/app/libs/landing/services/event-converter.service';
+import { User } from 'src/app/libs/auth/models/user';
+import { AuthFacadeService } from 'src/app/libs/auth/store/auth/auth.facade';
+import { EntityWrapper } from 'src/app/libs/auth/models/entity-wraper';
+import { EntityStatus } from 'src/app/libs/auth/models/entity-status';
+
+@Injectable()
+export class EventDescriptionEffects {
+	@Effect()
+	public GetEventdescription: Observable<Action> = this.actions$.pipe(
+		ofType(EventDescriptionActionTypes.LoadEventDescription),
+		switchMap((action: LoadEventDescription) =>
+			this.eventsRepository.getEvents().pipe(
```

Comment:
create route at backend which gets public version of event by id.
like `api/public-events/:id`